package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "kcdraidgroup",
+  "private": true,
+  "description": "Find Target 🎯 Communicate 📡 Raid ⚔️ Push Code 🎉",
+  "license": "UNLICENSED",
```

Comment:
Wasn't sure about this.  None of the "usual" OSS licenses made sense to me for this repo.  Not sure if there is one of the more artistic or non-software based licenses that would work better. 

Happy to take advice on this.

Reply:
🤔